### PR TITLE
Add argument valency check back in, with a whitelist.

### PR DESCRIPTION
**Pull request by jacqueswww**
### What I did
Fixes valency check todo.

### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: vyper/parser/lll_node.py [hunk 1]
```diff
@@ -91,9 +91,15 @@ def __init__(self,
...
-                    # TODO: remove dead code.
-                    # if arg.valency == 0:
-                    #     raise Exception("Can't have a zerovalent argument to an opcode or a pseudo-opcode! %r: %r" % (arg.value, arg))  # noqa: E501
+                    # pop and pass are used to push/pop values on the stack to be
+                    # consumed for private functions, therefore we whitelist this as a zero valency
+                    # allowed argument.
+                    zero_valency_whitelist = {'pass', 'pop'}
+                    if arg.valency == 0 and arg.value not in zero_valency_whitelist:
+                        raise Exception(
+                            "Can't have a zerovalent argument to an "
```
Comment (fubuloubu) [inline vyper/parser/lll_node.py]: Please, please, please a better error message here lol

Comment (jacqueswww) [reply vyper/parser/lll_node.py]: This is an internal API, I don't think we can make it any more clear? Any suggestions?

Comment (jacqueswww) [reply vyper/parser/lll_node.py]: I think we can just add, 'please file a bug report' ?

Comment (jacqueswww) [reply vyper/parser/lll_node.py]: @fubuloubu Added: file a bug report.

**Pull request merged**